tf_agents/environments/atari_wrappers.py

```diff
@@ -94,3 +94,19 @@ def _step(self, action):
   @property
   def game_over(self):
     return self._num_steps >= self._duration or self.gym.game_over
+
+
+class FireOnReset(gym.Wrapper):
+  """Start every episode with action 1 (FIRE) + another action (2).
+
+  This is required by some environments (e.g., Breakout) to actually start the
```

Comment:
It's not actually required - a DQN agent can solve breakout perfectly well without this wrapper! Especially if you evaluate with an epsilon-greedy policy (with small epsilon). So I'd reword this to mention that it can be helpful in the 3 games you mention, but that it is not completely necessary.